jrcc-access-spring-boot-autoconfigure/src/test/java/ca/bc/gov/open/jrccaccess/autoconfigure/AccessPropertiesTests.java

```diff
@@ -0,0 +1,49 @@
+package ca.bc.gov.open.jrccaccess.autoconfigure;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import ca.bc.gov.open.jrccaccess.autoconfigure.common.Constants;
+import ca.bc.gov.open.jrccaccess.autoconfigure.AccessProperties;
+import ca.bc.gov.open.jrccaccess.autoconfigure.AccessProperties.PluginConfig;
+
+import static org.junit.Assert.assertEquals;
+
+public class AccessPropertiesTests {
+
+	@Test
+	public void with_sender_should_set_sender() {
+		String sender = "bcgov";
+
+        AccessProperties properties = new AccessProperties();
+        properties.setInput(new AccessProperties.PluginConfig());
+        PluginConfig pluginConfig = properties.getInput();
+
+		pluginConfig.setSender(sender);
```

Comment:
not sure why the spacing is all funky again... is fine on my editor 😕 going to ignore this

Reply:
yeah something to do with github